ci/licenses_golden/excluded_files

```diff
@@ -41,6 +41,7 @@
 ../../../flutter/display_list/effects/dl_path_effect_unittests.cc
 ../../../flutter/display_list/geometry/dl_region_unittests.cc
 ../../../flutter/display_list/geometry/dl_rtree_unittests.cc
+../../../flutter/display_list/skia/dl_sk_canvas_unittests.cc
```

Comment:
Just noticed this - these tests can go in the existing dl_sk_conversions_unittests.cc which combines all of the tests for all of the conversions in all of the skia files.

Reply:
Sure thing - I was trying to follow the (inferred) convention of `foo.cc` has `foo_unittests.cc`.